src/plugins/intel_cpu/src/nodes/gather.cpp

```diff
@@ -580,6 +603,57 @@ void Gather::exec1DCase() {
     }
 }
 
+void Gather::execFusedCase() {
+    DEBUG_LOG(getName(), " execFusedCase");
+    auto srcMemPtr = getParentEdgeAt(GATHER_DATA)->getMemoryPtr();
+    auto idxMemPtr = getParentEdgeAt(GATHER_INDICES)->getMemoryPtr();
+    const auto* psrc = reinterpret_cast<const uint8_t*>(srcMemPtr->getData());
+    const auto* pidx = reinterpret_cast<const int32_t*>(idxMemPtr->getData());
+
+    const auto* zp = reinterpret_cast<const float_t*>(decompressionSubtractPtr->getData());
+    const auto* scale = reinterpret_cast<const float_t*>(decompressionMultiplyPtr->getData());
+
+    auto* pdst = reinterpret_cast<float_t*>(getChildEdgeAt(0)->getMemoryPtr()->getData());
+
+    const auto& idxDims = idxMemPtr->getStaticDims();
+    const auto batch = (idxDims.size() == 0) ? 1 : idxDims[0];
+    const auto seqLen = (idxDims.size() == 0) ? 1 : idxDims[1];
```

Comment:
`idxDims.size() == 0` => `idxDims.size() < 2` ?

Reply:
More strict condition had been added, so remove this judgement.